pkg/opensearch/client/search_request_test.go

```diff
@@ -88,9 +89,9 @@ func TestSearchRequest(t *testing.T) {
 						So(err, ShouldBeNil)
 						So(json.Get("size").MustInt(0), ShouldEqual, 200)
 
-						sort := json.GetPath("sort", timeField)
-						So(sort.Get("order").MustString(), ShouldEqual, "desc")
-						So(sort.Get("unmapped_type").MustString(), ShouldEqual, "boolean")
+						sort := json.Get("sort").MustArray()
+						So(sort, ShouldHaveLength, 1)
+						So(sort[0], ShouldResemble, map[string]interface{}{timeField: map[string]interface{}{"order": "desc", "unmapped_type": "boolean"}})
```

Comment:
should the types here be `map[string]string` instead of `map[string]interface{}`?

Reply:
This tests the type that comes from simplejson and also uses some "ShouldResemble" qualifier. It doesn't want to be more precise than interface{}.

Since we just care about the json, I moved the test to the bottom of the file and tested the json string equality.